styles.css

```diff
@@ -131,15 +128,428 @@
   display: block;
 }
 
+body {
+  /* overflow: hidden; */
+}
+
+main {
+  background-color: #e5e5e5;
+}
+
+.work-projects {
+  width: 327px;
+  height: auto;
+  margin: 100px 24px;
+  border-radius: 8px;
+  display: grid;
+  background-color: #fff;
+  gap: 1px;
+  grid-template-areas: 'header header header' 'description description description' 'footer footer footer';
+}
```

Comment:
According to the project requirements, you are supposed to use CSS grid in the main container of the works section and CSS  flexbox to layout each individual card. In your current implementation, you use CSS grid to layout individual cards. Please update your project to use grid to style the works section and flexbox to style each card🙏

<img width="775" alt="image" src="https://github.com/mhassa204/Portfolio-Project/assets/37116322/6ed42329-f53e-45b6-87e8-379191b82600">

Reply:
@Shisui6 Could you please help me with this part

Reply:
fixed!